src/test/java/bio/terra/MediaTypeUtilsTest.java

```diff
@@ -0,0 +1,75 @@
+package bio.terra;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+
+import bio.terra.cloudfunctions.utils.MediaTypeUtils;
+import bio.terra.common.BaseTest;
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.util.logging.Logger;
+import org.apache.commons.compress.archivers.ArchiveEntry;
+import org.apache.commons.compress.archivers.ArchiveInputStream;
+import org.apache.commons.compress.compressors.CompressorInputStream;
+import org.junit.Test;
+
+public class MediaTypeUtilsTest extends BaseTest {
+  private static final Logger logger = Logger.getLogger(MediaTypeUtilsTest.class.getName());
+
+  @Test
+  public void mockTGZTest() {
+    CompressorInputStream cis = null;
+    ArchiveInputStream ais = null;
+    BufferedInputStream bis = null;
+    try {
+      cis = MediaTypeUtils.createCompressorInputStream(MOCK_TGZ);
+      bis = new BufferedInputStream(cis);
+      ais = MediaTypeUtils.createArchiveInputStream(cis);
+      ArchiveEntry archiveEntry;
+      logger.info("mockTGZTest:");
+      int numberOfFilesProcessed = 0;
+      while ((archiveEntry = ais.getNextEntry()) != null) {
+        if (!archiveEntry.isDirectory()) {
+          numberOfFilesProcessed++;
+          verifyMockTGZArchiveEntry(archiveEntry.getName(), archiveEntry.getSize());
+        }
+      }
+      assertEquals(3, numberOfFilesProcessed);
```

Comment:
nit: add helpful messages to assertions, so we don't end up with inscrutable test failures like "2 was not equal to 3":
```
assertEquals("unexpected number of archive entries processed", 3, numberOfFilesProcessed);
```

Reply:
Added assertion message.